app/modules/user/feature_status.rb

```diff
@@ -24,6 +24,14 @@ def paypal_disconnect_allowed?
         !active_preorders?(charge_processor_id: PaypalChargeProcessor.charge_processor_id)
     end
 
+    def can_setup_bank_payouts?
+      active_bank_account.present? || native_payouts_supported? || signed_up_from_united_arab_emirates?
+    end
+
+    def can_setup_paypal_payouts?
+      payment_address.present? || !native_payouts_supported? || signed_up_from_united_arab_emirates? || signed_up_from_egypt?
```

Comment:
The only change is addition of `signed_up_from_egypt?` condition here. Rest of the conditions are just extracted out to these method.